bot/cogs/doc.py

```diff
@@ -150,16 +175,37 @@ def __init__(self, bot: commands.Bot):
         """
         self.base_urls[package_name] = base_url
 
-        fetch_func = functools.partial(intersphinx.fetch_inventory, config, '', inventory_url)
-        for _, value in (await self.bot.loop.run_in_executor(None, fetch_func)).items():
-            # Each value has a bunch of information in the form
-            # `(package_name, version, relative_url, ???)`, and we only
-            # need the relative documentation URL.
-            for symbol, (_, _, relative_doc_url, _) in value.items():
-                absolute_doc_url = base_url + relative_doc_url
-                self.inventories[symbol] = absolute_doc_url
-
-        log.trace(f"Fetched inventory for {package_name}.")
+        package = await self._fetch_inventory(inventory_url, config)
+        if package:
+            for group, value in package.items():
+                # Each value has a bunch of information in the form
+                # `(package_name, version, relative_url, ???)`, and we only
+                # need the package_name and the relative documentation URL.
+                for symbol, (package_name, _, relative_doc_url, _) in value.items():
```

Comment:
The code can already provide the information that your comment gives, so if you want to show that `version` is one of the items ignored, just give it a name with the `_` prefix and remove the unnecessary comments:
```py
        for group, value in package.items():
            for symbol, (package_name, _version, relative_doc_url, _) in value.items():
```

Reply:
This comment wasn't made by me, should I remove it anyway?

Reply:
If you agree with the comment, yeah it would fit under our idea of kaizen, improving upon aspects we work on that might not be immediately required under the scope of the PR.